EIPS/eip-7069.md

```diff
@@ -80,22 +78,23 @@ Execution semantics of `EXT*CALL`:
 11. Perform the call with the available gas and configuration.
 12. Push a status code on the stack:
     - `0` if the call was successful.
-    - `1` if the call has reverted (also can be pushed earlier in a light failure scenario).
-    - `2` if the call has failed.
+    - `1` if the call has reverted (also can be pushed earlier in a light failure scenario described in step 10).
+    - `2` if the call has failed (in case of general OOG failure or any scenario where all gas passed to the callee is burnt in case of an error).
 13. Gas not used by the callee is returned to the caller.
 
 Execution semantics of `RETURNDATALOAD`:
 
-1. Charge `G_verylow` (3) gas
+1. Charge `3` gas
 2. Pop 1 item from the stack, to be referred to as `offset`
 3. Push 1 item onto the stack, the 32-byte word read from the returndata buffer starting at `offset`.
 4. If `offset + 32 > len(returndata buffer)`, the result is zero-padded.
 
-In case this EIP is included as part of the greater EOF upgrade, execution semantics of `RETURNDATACOPY` in EOF formatted code ([EIP-3540](./eip-3540.md)) is modified as follows:
+Execution semantics of `RETURNDATACOPY` in EOF formatted code ([EIP-3540](./eip-3540.md)) is modified as follows:
 
 1. Assume the 3 arguments popped from stack are `destOffset`, `offset` and `size`.
```

Comment:
for points 1 and 2 here maybe mark it as "no change", to be explicit about where the semantics are modified?